frontend/src/components/atoms/TextField.tsx

```diff
@@ -1,8 +1,51 @@
 import React from "react";
+import { TextField } from "@mui/material";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
 
-/** A TextField page */
-const TextField = () => {
-  return <>Hello there</>;
+interface Props {
+  label: string;
+  required: boolean;
+  status: string; //error | ""
+  //if required, have * next to entry ?
+  //if error, display incorrect entry text
+  //onChange:
+  incorrectEntryText: string;
+}
+
+const theme = createTheme({
+  // palette: {
+  //   primary: {
+  //     main: "#8D8D8D",
+  //   },
+  //   secondary: {
+  //     main: "#D9D9D9",
+  //   },
+  // },
+});
```

Comment:
Comments like this can be distracting. Is this achieving anything? If not, kindly remove it